app/controllers/catalog_controller.rb

```diff
@@ -262,8 +262,8 @@ def self.modified_field
     config.add_sort_field "title_ssi desc, score desc", label: "title \u25BC"
     config.add_sort_field "issue_number_ssi desc", label: "issue number \u25BC"
     config.add_sort_field "issue_number_ssi asc", label: "issue number \u25B2"
-    config.add_sort_field "#{uploaded_field} desc", label: "date uploaded \u25BC"
-    config.add_sort_field "#{uploaded_field} asc", label: "date uploaded \u25B2"
+    config.add_sort_field "date_created_ssi desc", label: "date published \u25BC"
+    config.add_sort_field "date_created_ssi asc", label: "date published \u25B2"
```

Comment:
We might want this to be `date_created_dtsi` instead of `date_created_ssi`.  The way that solr sorts dates might be slightly different from how it sorts strings.

For example, if you have 3 dates like this:
`'1970-10', '1970-01', '1970-1'`
Will they sort in the expected order if they are strings instead of dates?

It will partially depend on how normalized your data is.

We can discuss it in more detail tomorrow.